cmd/test/main.go

```diff
@@ -207,6 +210,10 @@ func validateK8sVersionFormatIfProvided(k8sVersion string) error {
 	}
 }
 
+func WereViolationsFound(validationManager *ValidationManager, results *evaluation.ResultType) bool {
```

Comment:
I suggest to place it at the bottom and rename it to `wereViolationsFound` with lower case `w`, this way it will be like a private function

Reply:
And also, how about validating the opposite and rename it to "invocationsPassed"? @MentalBrake @hadar-co

Reply:
> And also, how about validating the opposite and rename it to "invocationsPassed"? @MentalBrake @hadar-co

@noaabarki since the validation is in sync with the error we are returning logic-wise, I think we should leave it as is, it is much clearer. I will move the function to the bottom as you suggested :)